multimodal/src/autogluon/multimodal/problem_types.py

```diff
@@ -0,0 +1,221 @@
+"""Problem types supported in MultiModalPredictor"""
+
+from dataclasses import dataclass, field
+from typing import Optional, Set
+
+from .constants import (
+    BINARY,
+    CATEGORICAL,
+    CLASSIFICATION,
+    FEATURE_EXTRACTION,
+    FEW_SHOT_TEXT_CLASSIFICATION,
+    IMAGE,
+    IMAGE_BYTEARRAY,
+    IMAGE_SIMILARITY,
+    IMAGE_TEXT_SIMILARITY,
+    MULTICLASS,
+    NAMED_ENTITY_RECOGNITION,
```

Comment:
I do not recall I used this constant: "NAMED_ENTITY_RECOGNITION". Shall we use NER only?

Reply:
Both `NER` and `NAMED_ENTITY_RECOGNITION` will be mapped to `NER`.

Reply:
You can view `NAMED_ENTITY_RECOGNITION` as the alias of `NER` problem type.